internal/clientcache/cmd/daemon/start.go

```diff
@@ -162,16 +167,24 @@ func (c *StartCommand) Run(args []string) int {
 	// report if the daemon started or not.
 
 	logFilePath := filepath.Join(dotDir, logFileName)
```

Comment:
perhaps a comment that explains how this will work with the eventing.  it's a
bit of state which is easy to explain now... and future devs might appreciate
it.

Also, can you add a unit test for this (a new log file and it's rotation)?